src/main/java/flaxbeard/steamcraft/client/render/BlockSteamPipeRenderer.java

```diff
@@ -30,10 +30,10 @@
     public static boolean updateWrenchStatus() {
         EntityPlayer player = Minecraft.getMinecraft().thePlayer;
         Item equipped = player.getCurrentEquippedItem() != null ? player.getCurrentEquippedItem().getItem() : null;
-        if (equipped != null) {
-        	 return (player.getHeldItem() != null && player.getHeldItem().getItem() instanceof IPipeWrench && ((IPipeWrench) equipped).canWrench(player, (int) player.posX, (int) player.posY, (int) player.posZ) && !player.isSneaking());
-        }
-        return false;
+        Item held = player.getHeldItem().getItem();
+        return equipped != null && held != null && held instanceof IPipeWrench
+                && ((IPipeWrench) equipped).canWrench(player, (int) player.posX, (int) player.posY, (int) player.posZ)
+                && !player.isSneaking();
```

Comment:
Either equipped and held are actually the same thing, or this will cause a ClassCastException. I'll have to investigate `EntityPlayer#getCurrentEquippedItem` and `EntityPlayer#getHeldItem`.